source/includes/_examples.md

```diff
@@ -0,0 +1,219 @@
+# Examples
+
+## CDC Guideline for Prescribing Opioids for Chronic Pain - Request
+
+```json
+{
+  "hookInstance": "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
+  "fhirServer": "http://fhirtest.uhn.ca/baseDstu2",
```

Comment:
Rather than use a real FHIR server URL here (and below), would you rather use something like https://fhir.example.com? I know we use http://hooks.smarthealthit.org:9080 in another example in the doc but these URLs will likely become broken in the future and developers may have expectations that they can hit this URL in the example. Using the example.com domain makes it clear that these are pure examples.

Thoughts?